models/majorModel.js

```diff
@@ -1,8 +1,51 @@
-var mongoose = require('mongoose');
+const mongoose = require('mongoose');
 
-// Create major Schema
-var majorSchema = mongoose.Schema({
-  sample: String,
+// Create a Major Schema
```

Comment:
same thing I told Logan here, this comment isn't really necessary because I can tell what you're doing here from the code.